.rubocop.yml

```diff
@@ -46,6 +46,9 @@ Layout/ParameterAlignment:
 Lint/AmbiguousBlockAssociation:
   Enabled: false
 
+Lint/IncompatibleIoSelectWithFiberScheduler:
```

Comment:
Fiber-aware scheduling, instead of ths
`ready_sockets, = IO.select([@server])`
this:
`@server.wait_readable`

But afaik Ruby < 3, doesn't have fiber-aware scheduling or `wait_readable`, so I've disabled this linter.